snap/snapcraft.yaml

```diff
@@ -67,6 +67,8 @@ apps:
   kubectl:
     command: microk8s-kubectl.wrapper
     completer: kubectl.bash
+  version:
```

Comment:
With this you add a new command. You will be able to call:
```
microk8s version
```
If I understand this correctly we want to add a `--version` argument to the already existing `microk8s` command.